app/components/help/form_component.html.erb

```diff
@@ -1,12 +1,12 @@
-<div class="modal fade" id="contactUsModal" tabindex="-1" aria-labelledby="content-type-prompt" aria-hidden="true">
+<div class="modal fade" id="contactUsModal" tabindex="-1" aria-labelledby="help-prompt" aria-hidden="true">
   <div class="modal-dialog modal-dialog-centered ">
     <div class="modal-content">
       <div class="modal-header">
-        <h3 class="modal-title">Help</h3>
+        <h3 class="modal-title" id="help-prompt">Help</h3>
         <button type="button" class="btn-close" data-dismiss="modal" aria-label="Close"></button>
       </div>
       <div class="modal-body" data-controller="help">
-        <%= form_with(url: "#{help_path}.json",
+        <%= form_with(url: "#{help_path}",
```

Comment:
Can you use `help_path` here instead of `"#{help_path}"` or is the extra string interpolation necessary?

Reply:
Updated.